instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb

```diff
@@ -15,6 +15,45 @@ module Patches
         module Client
           def query(sql, options = {})
             attributes = client_attributes
+            attributes.merge!(span_attributes(sql))
+
+            tracer.in_span(
+              span_name(sql),
+              attributes: attributes.merge!(OpenTelemetry::Instrumentation::Mysql2.attributes),
+              kind: :client
+            ) do
+              super(sql, options)
+            end
+          end
+
+          def prepare(sql)
+            attributes = client_attributes
+            attributes.merge!(span_attributes(sql))
+
+            tracer.in_span(
+              span_name(sql),
+              attributes: attributes.merge!(OpenTelemetry::Instrumentation::Mysql2.attributes),
+              kind: :client
+            ) do
+              super(sql)
+            end
+          end
+
+          private
+
+          def span_name(sql)
+            OpenTelemetry::Helpers::MySQL.database_span_name(
+              sql,
+              OpenTelemetry::Instrumentation::Mysql2.attributes[
+                SemanticConventions::Trace::DB_OPERATION
+              ],
+              database_name,
+              config
+            )
+          end
+
+          def span_attributes(sql)
```

Comment:
If you're able to update this as well please

Reply:
Thanks for the review, I have changed all the private method with `_otel`, but let me if some of the private method is not required with `_otel`.